### PR TITLE
conv shift op: change to CamelCase & fix bug

### PR DESCRIPTION
**Pull request by mkliegl**
Addressing part of issue #5549 .

### CODE REVIEW

**Comment by mkliegl** [comment]
@chengduoZH The tests pass now after rebasing. Could you please take a look?

File: paddle/operators/conv_shift_op.cu [hunk 1]
```diff
@@ -33,9 +33,9 @@ inline int div_up(int x, int y) { return (x + y - 1) / y; }
...
 template <typename T>
-__global__ void conv_shift_forward(const T *x, const T *y, T *out, int x_width,
-                                   int y_width, int y_half_width,
-                                   int batch_size) {
+__global__ void ConvShiftForward(const T *x, const T *y, T *out, int x_width,
+                                 int y_width, int y_half_width,
+                                 int batch_size) {
   extern __shared__ T mem[];
 
   int tx = threadIdx.x;
```
Comment (chengduoZH) [inline paddle/operators/conv_shift_op.cu]: Although the problem mentioned in issue has been solved, there is a problem with the ConvShiftForward function.
Please recheck line 62~68, you should not use `return` before `__syncthreads();`

**Comment by chengduoZH** [comment]
@mkliegl 
I can approve it, but you should write a new PR to solve above problem. Because the problem is a little bit serious.

**Comment by chengduoZH** [comment]
If some functions are used in Op, the functions are generally written in the form of functor, such as [LSTM](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/operators/lstm_op.h#L59).

File: paddle/operators/conv_shift_op.cu (hunk 1 above)
Comment (mkliegl) [reply paddle/operators/conv_shift_op.cu]: Good catch. Let me fix this now quickly.

**Comment by mkliegl** [comment]
@chengduoZH Thank you for your review! I added a fix for the issue you pointed out now rather than making another PR. Could you please check one more time?

Regarding making functors: It doesn't seem likely that functors for the conv shift forward and backward kernels are going to be needed anywhere else, so I'm not sure it is worth making this change now. But if you still prefer this for consistency or other reasons, let me know and I can do it. Or else please feel free to just make the changes yourself.

File: paddle/operators/conv_shift_op.cu [hunk 2]
```diff
@@ -33,9 +33,9 @@ inline int div_up(int x, int y) { return (x + y - 1) / y; }
 // y is fairly small. For large y, it would probably be more efficient
 // to also tile across y.
 template <typename T>
-__global__ void conv_shift_forward(const T *x, const T *y, T *out, int x_width,
-                                   int y_width, int y_half_width,
-                                   int batch_size) {
+__global__ void ConvShiftForward(const T *x, const T *y, T *out, int x_width,
+                                 int y_width, int y_half_width,
+                                 int batch_size) {
```
Comment (chengduoZH) [inline paddle/operators/conv_shift_op.cu]: Please reorder parameter.
[Function_Parameter_Ordering](https://google.github.io/styleguide/cppguide.html#Function_Parameter_Ordering)
The following function parameters are the same.

File: paddle/operators/conv_shift_op.cu [hunk 3]
```diff
@@ -160,20 +160,20 @@ class ConvShiftGradKernel<platform::GPUPlace, T>
     auto stream = context.cuda_device_context().stream();
 
     const int x_per_block = 256;
-    int num_x_blocks = div_up(x_width, x_per_block);
+    int num_x_blocks = DivUp(x_width, x_per_block);
     dim3 grid_dim(num_x_blocks, y_width, batch_size);
 
     if (dX) {
       T *dx_data = dX->mutable_data<T>(context.GetPlace());
       cudaMemsetAsync(dx_data, 0, dX->numel() * sizeof(T), stream);
```
Comment (chengduoZH) [inline paddle/operators/conv_shift_op.cu]: You can clean memory in [this way](https://github.com/PaddlePaddle/Paddle/blob/2f3d1c345ff552678a2166e1e724e0af320cd427/paddle/operators/lstm_op.h#L200).

**Comment by chengduoZH** [comment]
@mkliegl 
In that case, I think you can write it in this way(not define functor) now. If these functions are needed in other Op, we can change them.

**Pull request merged**